src/selectors.js

```diff
@@ -2,8 +2,18 @@ import { matchPath } from "react-router"
 
 const createSelectors = (structure) => {
   const { getIn, toJS } = structure
-  const getLocation = state => toJS(getIn(state, ['router', 'location']))
-  const getAction = state => toJS(getIn(state, ['router', 'action']))
+
+  const _getLocation = state => getIn(state, ['location'])
+  const _getAction = state => getIn(state, ['action'])
```

Comment:
I think we don't need this because it doesn't do anything beside getting a key at only one level depth.